backend/chromeApiServer/src/test/java/com/canvas/service/helperServices/AESCryptoServiceTest.java

```diff
@@ -0,0 +1,58 @@
+package com.canvas.service.helperServices;
+import com.canvas.exceptions.CanvasAPIException;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.springframework.core.env.Environment;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import static org.mockito.Mockito.*;
+
+
+public class AESCryptoServiceTest {
+
+    @Mock
+    Environment env;
+
+    @Before
+    public void init(){
+        env = mock(Environment.class);
+        when(env.getProperty(any()))
+                .thenReturn("someSecret");
+    }
+
+    @Test
+    public void testEncryptAndDecrypt() {
```

Comment:
This is not a descriptive test name, what are you testing and expecting? Applies to all test names in this class

Reply:
wil change the test name